service/src/test/java/com/myhome/security/controllers/CommunityControllerTest.java

```diff
@@ -0,0 +1,45 @@
+package com.myhome.security.controllers;
+
+import com.myhome.controllers.integration.ControllerIntegrationTestBase;
+import com.myhome.controllers.request.AddAmenityRequest;
+import com.myhome.controllers.request.AddCommunityAdminRequest;
+import com.myhome.controllers.request.LoginUserRequest;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.*;
+
+import javax.servlet.http.HttpServletResponse;
+import java.util.HashSet;
+import java.util.Set;
+
+import static java.util.Collections.singletonList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class CommunityControllerTest  extends ControllerIntegrationTestBase {
+
+    private final String addAmenityUrl = "communities/19dfd456-5a63-4724-86c5-73ff53724c3a/amenities";
+    private final String addCommunityAdminUrl = "communities/19dfd456-5a63-4724-86c5-73ff53724c3a/admins";
+    private final String email = "test@test.com";
+    private final String password = "testtest";
+
+    @Test
+    public void addAmenityToCommunityWithNoAdminUser() {
+        updateJwtToken(new LoginUserRequest(email, password));
+        ResponseEntity<String> responseEntity = sendRequest(HttpMethod.POST, addAmenityUrl, null);
+
+        assertEquals(HttpServletResponse.SC_UNAUTHORIZED, responseEntity.getStatusCodeValue());
+    }
+
+    @Test
+    public void addAmenityToCommunityWithAdminUser() {
+        updateJwtToken(new LoginUserRequest(email, password));
```

Comment:
Could you please use given/when/then pattern?
You can look on `HouseMemberDocumentServiceIntegrationTest` class to see how it is made in tests